src/components/studies/participants/PageBox.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import { Button } from '@material-ui/core'
+
+const useStyles = makeStyles(theme => ({
+  pageBoxSurrounding: {
+    cursor: 'pointer',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '25px',
+    width: '25px',
+    minWidth: '0px',
+    borderRadius: '0px',
+  },
+}))
+
+const PageBox: React.FC<{
```

Comment:
For consistency  let's use React.FunctionComponent, declare the type of the parameters and deconstruct them.

So this would look something like:
```
type PageBoxProps = {
isSelected: boolean
  pageNumber: number
  onPageSelected: Function
}
const PageBox: React.FunctionComponent<PageBoxProps> = ({
  isSelected: boolean
  pageNumber: number
  onPageSelected: Function}) => ...
```